Makefile

```diff
@@ -238,7 +259,9 @@ bench: ## Run benchmarks.
 bench: TESTTIMEOUT := $(BENCHTIMEOUT)
 
 .PHONY: check test testshort testrace testlogic bench
+test: ## Run tests.
 check test testshort testrace bench: gotestdashi
+	$(warning You specified TESTS=$(TESTS) without specifying PKG - your tests will run against all packages)
```

Comment:
Holdover from an earlier attempt? This doesn't seem to check if `PKG` is actually empty.